templates/base.html

```diff
@@ -20,6 +20,7 @@
     <title>{% block "title" %}{% endblock "title" %}Where to find me</title>
 
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/fork-awesome@1.1.7/css/fork-awesome.min.css" integrity="sha256-gsmEoJAws/Kd3CjuOQzLie5Q3yshhvmo7YNtBG7aaEY=" crossorigin="anonymous">
```

Comment:
Mild preference for leaving a note as to which icons this adds or linking to any discussions on fontawesome explaining what’s going on and why this is forked, for posterity, but that’s not a good reason to hold this up.